bundles/org.openhab.binding.sncf/README.md

```diff
@@ -0,0 +1,70 @@
+# SNCF Binding
+
+The SNCF binding provides real-time data for each train, bus, tramway... station in France.
+This is based on live API provided by DIGITALSNCF.
+
+Get you API key at https://www.digital.sncf.com/startup/api/token-developpeur
+
+Note : SNCF Api is based on the open [API Navitia](https://doc.navitia.io/#getting-started). This binding uses a very small subset of it, restricted to its primary purpose.
+
+## Supported Things
+
+Bridge: The binding supports a bridge to connect to the [DIGITALSNCF service](https://www.digital.sncf.com/startup/api/token-developpeur). A bridge uses the thing ID "api".
+
+Station: Represents a given bus, train station.
+
+Of course, you can add multiple as many stations as needed.
+
+
+## Discovery
+
+This binding takes care of auto discovery. This method is strongly recommended as its the only way to get proper station ID depending upon transportation type.
+
+To enable auto-discovery, your location system setting must be defined. Once done, at first launch discovery will search every station in a radius of 2500 m around the system, extending it by step of 500 m until it finds a first set of results.
```

Comment:
I like this idea.
Please one line per sentence.
I think a comma is missing just after "launch".
Looking at the code, my feeling is that the initial radius is 2000 m (1500+500). To be clarified.